From: Dave Reisner <dreisner@archlinux.org>
Date: Sun, 13 Oct 2013 17:42:51 -0400
Subject: udevadm.xml: document --resolve-names option for test

And remove documentation of the --subsystem flag which doesn't actually
exist.
---
 man/udevadm.xml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/man/udevadm.xml b/man/udevadm.xml
index d0b257de..b959216d 100644
--- a/man/udevadm.xml
+++ b/man/udevadm.xml
@@ -462,9 +462,13 @@
           </listitem>
         </varlistentry>
         <varlistentry>
-          <term><option>--subsystem=<replaceable>string</replaceable></option></term>
+          <term><option>--resolve-names=<replaceable>early|late|never</replaceable></option></term>
           <listitem>
-            <para>The subsystem string.</para>
+            <para>Specify when udevadm should resolve names of users and groups.
+              When set to early (the default) names will be resolved when the
+              rules are parsed. When set to late names will be resolved for
+              every event. When set to never names will never be resolved and
+              all devices will be owned by root.</para>
           </listitem>
         </varlistentry>
         <varlistentry>
